yesod-core/Yesod/Core/Types.hs

```diff
@@ -250,6 +250,10 @@ instance (a ~ (), Monad m) => Monoid (WidgetT site m a) where
     mempty = return ()
     mappend x y = x >> y
 instance (a ~ (), Monad m) => Semigroup (WidgetT site m a)
+instance Monad m => IsString (WidgetT site m ()) where
```

Comment:
I only remembered because the previous instance uses it

On Tue, Jul 21, 2015, 8:36 AM Andrew Martin <notifications@github.com>
wrote:

> In yesod-core/Yesod/Core/Types.hs
> <https://github.com/yesodweb/yesod/pull/1038#discussion_r35116687>:
>
> > @@ -250,6 +250,10 @@ instance (a ~ (), Monad m) => Monoid (WidgetT site m a) where
> >      mempty = return ()
> >      mappend x y = x >> y
> >  instance (a ~ (), Monad m) => Semigroup (WidgetT site m a)
> > +instance Monad m => IsString (WidgetT site m ()) where
>
> Ah! That dumb trick I always forget.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yesodweb/yesod/pull/1038/files#r35116687>.
>